demux/codec_tags.c

```diff
@@ -45,14 +45,55 @@ static const char *lookup_tag(int type, uint32_t tag)
 }
 
 // Corresponds to WMMEDIASUBTYPE_Base.
-static const unsigned char guid_ext_base[16] =
+static const unsigned char guid_ext_base[3][16] = {
     {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x10, 0x00,
-     0x80, 0x00, 0x00, 0xAA, 0x00, 0x38, 0x9B, 0x71};
+     0x80, 0x00, 0x00, 0xAA, 0x00, 0x38, 0x9B, 0x71},
+    {0x00, 0x00, 0x00, 0x00, 0x21, 0x07, 0xD3, 0x11,
+     0x86, 0x44, 0xC8, 0xC1, 0xCA, 0x00, 0x00, 0x00},
+    {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+     0x00, 0x00, 0x10, 0x00, 0x80, 0x00, 0x00, 0xAA}
```

Comment:
neither can I for 0000-0000-0000-1000800000aa
where do these come from?

Reply:
https://github.com/FFmpeg/FFmpeg/blob/master/libavformat/riff.h#L114